src/main/resources/assets/xpbook/lang/uk_ua.json

```diff
@@ -0,0 +1,7 @@
+{
+    "item.xpbook.xp_book": "(ЗАСТАРІЛО) - ПКМ, ЩОБ КОНВЕРТУВАТИ",
+    "item.xpbook.xp_tome": "Книга досвіду",
+    "xpbook.tooltip.1": "Shift + ПКМ, щоб зберегти якомога більше досвіду",
```

Comment:
Using "Shift" is ambiguous, because players can use a different key than their shift key in order to sneak. That's why the english language file says "*Sneak* + right-click", instead of "*Shift* + right-click".